src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -15,10 +15,20 @@ public class ConsoleLogger : ILogger
         private static readonly object _lock = new object();
         private static readonly string _loglevelPadding = ": ";
         private static readonly string _messagePadding;
+        private static readonly string _newLineWithMessagePadding;
 
         // ConsoleColor does not have a value to specify the 'Default' color
         private readonly ConsoleColor? DefaultConsoleColor = null;
 
+        // do not change user's background color except for Critical
+        private readonly ConsoleColors CriticalConsoleColors = new ConsoleColors(ConsoleColor.White, ConsoleColor.Red);
+        private readonly ConsoleColors ErrorConsoleColors = new ConsoleColors(ConsoleColor.Red, ConsoleColor.Black);
```

Comment:
Oh, I think the comment is outdated, we made a decision to set black as background for everything to make sure the foreground color was visible